prdoc/pr_4755.prdoc

```diff
@@ -0,0 +1,24 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Send PeerViewChange with high priority
+
+doc:
+  - audience: Node Operator
```

Comment:
I think the Node developers should be the main target audience. The node operators would like want to hear if and how would this reduces the CPU utilization of the node.